taglib/asf/asftag.h

```diff
@@ -184,8 +184,8 @@ namespace TagLib {
       AttributeList attribute(const String &name) const;
 
       /*!
-       * Sets the \a key attribute to the value of \a attribute. If an attribute
-       * with the \a key is already present, it will be replaced.
+       * Sets the \a attribute attribute to the value of \a attribute. If an attribute
```

Comment:
`attribute` here should be either `name` or `key`

Reply:
I think this is what you meant.

Reply:
I think it should be `Sets the \a name attribute to the value of \a attribute` since `name` is the dictionary key.

Reply:
Fixed